testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/CamundaProcessTestContext.java

```diff
@@ -93,4 +94,15 @@ public interface CamundaProcessTestContext {
    * @param timeToAdd the duration to add to the current time
    */
   void increaseTime(final Duration timeToAdd);
+
+  /**
+   * Creates a mock job worker for the specified job type.
+   *
+   * <p>This mock allows simulating job processing behavior, such as completing jobs, throwing BPMN
+   * errors, or handling jobs with custom logic.
+   *
+   * @param jobId the job type to mock, matching the `zeebeJobType` in the BPMN model.
+   * @return a `JobWorkerMock` instance for configuring the mock behavior.
```

Comment:
🔧 Use `{@see JobWorkerMock}`

```suggestion
   * @return a {@see JobWorkerMock} instance for configuring the mock behavior.
```